### DISCUSSION TITLE
how to make Toast UI calendar scrollable

### DISCUSSION THREAD

**Discussion by korijamal** [nhn/tui.calendar#1348] [Q&A] [answered]
## Version

v2.1.3

## Test Environment
Google Chrome, Mac OS, Next.js, Typescript

## Current Behavior

<img width="311" alt="Screenshot 2023-01-10 at 6 15 03 PM" src="https://user-images.githubusercontent.com/81694120/211510458-31d391b3-4c06-4282-8ea5-ad4bf9a8fdd7.png">
Code: 
<Calendar
                  ref={calendarRef}
                  height={height}
                  view="month"
                  isReadOnly={true}
                  calendars={calendars}
                  schedules={schedules}
                  usageStatistics={false}
                  month={{
                    daynames: ["일", "월", "화", "수", "목", "금", "토"],
                    scheduleFilter: function (schedule) {
                      console.log("schedule:" + schedule);
                      return schedule.isVisible;
                    },
                    visibleWeeksCount: 1,
                  }}
                />

// Write example code

## Expected Behavior

 I want to make it scrollable to right or left, but I could not find the method for it from the documentation. Is it even possible with Toast Calendar? Please help me with this issues
![image](https://user-images.githubusercontent.com/81694120/211512528-8991c01f-4b26-49c1-9948-b0781a67cd8d.png)

**Comment by adhrinae** [comment, marked as answer]
Duplicate of [#865](https://github.com/nhn/tui.calendar/issues/865#issuecomment-885328623).

Check out the comment in the link.

**Reply by korijamal** [reply]
I checked, and gave enough width, but it is not scrolling. Eventhough it worked, only current month will be scrollable. What I want to make is, move to next week or month, when u scroll right. Same move to last month or week when u scroll left

**Reply by adhrinae** [reply]
I see. TUI Calendar doesn't offer the feature you want.
You can leverage `ScrollEvent` to implement that feature.

Example: https://stackblitz.com/edit/js-oumecf?file=index.js (⚠️ Not optimized for production)
